### ISSUE TITLE
Toggle Integrated Terminal command doesn't close integrated terminal if it's open

### ISSUE THREAD

**Issue by schu34** [microsoft/vscode#79040]
Issue Type: <b>Bug</b>

1. Open integrated terminal
2. Open Command Pallet, enter `Toggle Integrated Terminal`

 

VS Code version: Code 1.37.0 (036a6b1d3ac84e5ca96a17a44e63a87971f8fcc8, 2019-08-08T01:22:37.660Z)
OS version: Darwin x64 18.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2900)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 2, 2|
|Memory (System)|32.00GB (9.76GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|10.6.0
react-pure-to-class-vscode|ang|1.1.8
vscode-eslint|dba|1.9.0
es7-react-js-snippets|dsz|2.4.0
gitlens|eam|9.9.3
prettier-vscode|esb|1.9.0
run-in-terminal|kor|0.0.1
vscode-scss|mrm|0.6.2
vscode-coverage-gutters|rya|2.4.0
code-spell-checker|str|1.7.17
vim|vsc|1.9.0

(1 theme extensions excluded)

</details>

**Comment by usernamehw** [comment]
I think it's `as-designed` after this issue: #19400. 

But it should be possible to configure keybindings to make it work differently: https://github.com/microsoft/vscode/issues/19400#issuecomment-423074454

**Comment by schu34** [comment]
This was very confusing to me, I thing just because of the naming. When I searched for `terminal` in keyboard shortcuts, all I could find was the toggle command. I wonder if a `workspace.actions.closeTerminal` action could be added as it's more discoverable. I realized now that I can use `workspace.actions.closePanel` but not sure how a normal user would know to look for that unless they quite a bit of time digging through github issues
